### PR TITLE
port aliyun target to net core

### PR DESCRIPTION
**Pull request by lust4life**

### CODE REVIEW

**Comment by lust4life** [comment]
@haf  cloud you help release this nuget pkg (https://www.nuget.org/packages/Logary.Targets.AliYun/) ?

File: src/targets/Logary.Targets.Stackdriver/App.config [hunk 1]
```diff
@@ -12,6 +12,21 @@
     <gcServer enabled="true" />
 
   <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
+  <dependentAssembly>
```
Comment (haf) [inline src/targets/Logary.Targets.Stackdriver/App.config]: Please remove this change to the StackDriver taget.

**Comment by haf** [comment]
Hi, yes I will but please revert the change to the stackdriver appconfig.

File: src/targets/Logary.Targets.Stackdriver/App.config (hunk 1 above)
Comment (lust4life) [reply src/targets/Logary.Targets.Stackdriver/App.config]: done.

**Pull request merged**